index.php

```diff
@@ -2,33 +2,25 @@
 date_default_timezone_set("Europe/Moscow");
 require_once('helpers.php');
 require_once('data.php');
+require_once('functions.php');
 
+/** Подключение к БД */
 
+$connection = do_connection();
 
+/** Получаем данные о пользователе */
 
+$user = find_users($connection);
```

Comment:
Т.е. здесь нужно будет передать user_id
```suggestion
$user_id = 1;

$user = find_users($connection, $user_id);
```